internal/go/skycfg/proto_enum.go

```diff
@@ -74,6 +77,24 @@ func (v *skyProtoEnumValue) Hash() (uint32, error) {
 	return starlark.MakeInt64(int64(v.value)).Hash()
 }
 
+func (v *skyProtoEnumValue) CompareSameType(op syntax.Token, y starlark.Value, depth int) (bool, error) {
+	other, ok := y.(*skyProtoEnumValue)
```

Comment:
Using upstream Starlark as an example, it seems that `CompareSameType` is allowed to assume its input `Value` is already of the correct type (and `panic` if this requirement isn't met).

See https://github.com/google/starlark-go/blob/0aa95694c768/starlark/value.go#L714-L715:
```go
func (x *Dict) CompareSameType(op syntax.Token, y_ Value, depth int) (bool, error) {
	y := y_.(*Dict)
	switch op {
	case syntax.EQL:
```
```

Reply:
Interesting. Done